test/pkg_autocorrects_test.cc

```diff
@@ -20,51 +20,72 @@ namespace spd = spdlog;
 auto logger = spd::stderr_color_mt("pkg-autocorrects-test");
 auto errorQueue = make_shared<sorbet::core::ErrorQueue>(*logger, *logger);
 
+string examplePackage = "class Opus::ExamplePackage < PackageSpec\nend\n";
+string examplePackagePath = "example/__package.rb";
```

Comment:
To catch the old issues, we now enter two packages into the global state, rather than the lazy shortcut being used before.